### ISSUE TITLE
How to run test.py

### ISSUE THREAD

**Issue by ykwang1207** [cabooster/UTOM#2]
Hi. I've downloaded all three given datasets for training and testing. The training process was successful. However when I tried the testing process as you mentioned in "readme.md", errors occured to warn me a necessity of directory "testB". Could you please guide me if there was something wrong with my operation or understanding?

**Comment by cabooster** [comment]
I think you just need to create a folder named `testB` and put it together with `trainA`, `trainB` and `testA`. `testB`  is the folder to store transformed images from `testA`

**Comment by ykwang1207** [comment]
@cabooster Thanks. I've tried to create folder testB, however another error occured when executing "index_B = index % self.B_size" in program "unaligned_dataset.py", indicating "ZeroDivisionError: integer division or modulo by zero". What shall I do then?

**Comment by runn2020** [comment]
Yes, I also encountered the same problem. I created folder testB, error occured when executing "index_B = index % self.B_size" in program "unaligned_dataset.py", indicating "ZeroDivisionError: integer division or modulo by zero".

**Comment by runn2020** [comment]
> @cabooster Thanks. I've tried to create folder testB, however another error occured when executing "index_B = index % self.B_size" in program "unaligned_dataset.py", indicating "ZeroDivisionError: integer division or modulo by zero". What shall I do then?
Have you solved it yet? Can you help me

**Comment by YinniKun** [comment]
@ykwang1207 @runn2020 you can just put random stuff in testB if you are not intended on using it, so that the length of testB is not zero
